src/foam/nanos/notification/sms/TwilioSMSService.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.notification.sms',
+  name: 'TwilioSMSService',
+
+  implements: [
+    'foam.nanos.notification.sms.SMSService'
+  ],
+
+  documentation: 'Sends SMS message through Twilio',
+
+  javaImports: [
+    'com.twilio.rest.api.v2010.account.Message',
+    'com.twilio.Twilio',
+    'com.twilio.type.PhoneNumber',
+    'foam.nanos.app.TwilioConfig',
+    'foam.nanos.notification.sms.SMSStatus',
+    'foam.nanos.om.OMLogger',
+    'foam.util.SafetyUtil'
+  ],
+
+  methods: [
+    {
+      name: 'sendSms',
+      javaCode: `
+        OMLogger omLogger = (OMLogger) x.get("OMLogger");
+        TwilioConfig twilioConfig = (TwilioConfig) x.get("twilioConfig");
+        Twilio.init(twilioConfig.getAccountSid(), twilioConfig.getAuthToken());
```

Comment:
validate that the SID and token are set .. if they are not, log an error message and return right away, leaving the message unsent.